go-controller/pkg/util/sync.go

```diff
@@ -30,3 +30,7 @@ func GetChildStopChanWithTimeout(parentStopChan <-chan struct{}, duration time.D
 func WaitForNamedCacheSyncWithTimeout(controllerName string, stopCh <-chan struct{}, cacheSyncs ...cache.InformerSynced) bool {
 	return cache.WaitForNamedCacheSync(controllerName, GetChildStopChanWithTimeout(stopCh, types.InformerSyncTimeout), cacheSyncs...)
```

Comment:
hmm do you think we should rename this util from 
`func WaitForNamedCacheSyncWithTimeout(controllerName string, stopCh <-chan struct{}, cacheSyncs ...cache.InformerSynced) bool {`
to 
`func WaitForInformerCacheSyncWithTimeout(controllerName string, stopCh <-chan struct{}, cacheSyncs ...cache.InformerSynced) bool {`

so that its explicit ? NamedCacheSync seems too generic?

Reply:
True

Reply:
resolved thanks!